quickstarts/microsoft.apimanagement/api-management-create-all-resources/main.bicep

```diff
@@ -0,0 +1,266 @@
+@description('The email address of the owner of the service')
+@minLength(1)
+param publisherEmail string
+
+@description('The name of the owner of the service')
+@minLength(1)
+param publisherName string
+
+@description('The pricing tier of this API Management service')
+@allowed([
+  'Consumption'
+  'Developer'
+  'Basic'
+  'Standard'
+  'Premium'
+])
+param sku string = 'Developer'
+
+@description('The instance size of this API Management service.')
+param skuCount int = 1
+
+@description('Tenant policy XML.')
+param tenantPolicy string
+
+@description('API policy XML.')
+param apiPolicy string
+
+@description('Operation policy XML.')
+param operationPolicy string
+
+@description('Product policy XML.')
+param productPolicy string
+
+@description('Location for all resources.')
+param location string = resourceGroup().location
+
+var apiManagementServiceName = 'apiservice${uniqueString(resourceGroup().id)}'
+
+resource apiManagementService 'Microsoft.ApiManagement/service@2022-08-01' = {
+  name: apiManagementServiceName
+  location: location
+  tags: {}
+  sku: {
+    name: sku
+    capacity: skuCount
+  }
+  properties: {
+    publisherEmail: publisherEmail
+    publisherName: publisherName
+  }
+}
+
+resource apiManagementServiceName_policy 'Microsoft.ApiManagement/service/policies@2022-08-01' = {
+  parent: apiManagementService
+  name: 'policy'
+  properties: {
+    value: tenantPolicy
+  }
+}
+
+resource apiManagementServiceName_PetStoreSwaggerImportExample 'Microsoft.ApiManagement/service/apis@2022-08-01' = {
```

Comment:
looks like all of the resource symbol names are autogenerated from the bicep decompiler. Can these be renamed to be more human friendly?